packages/preview/biz-report/0.1.0/typst.toml

```diff
@@ -0,0 +1,11 @@
+[package]
```

Comment:
I think it would be worth it to turn `example.typ` into a [template](https://github.com/typst/packages/blob/0488b38bb7fdfbce81981803ebbcf5612f8919f8/docs/manifest.md#templates) to let people quickly start a new project using your package.